velox/exec/tests/AssignUniqueIdTest.cpp

```diff
@@ -92,3 +92,26 @@ TEST_F(AssignUniqueIdTest, multiThread) {
     verifyUniqueId(plan, input, 8);
   }
 }
+
+TEST_F(AssignUniqueIdTest, maxRowIdLimit) {
+  auto input = {
+      makeRowVector({makeFlatVector<bool>(1, [](auto row) { return row; })})};
```

Comment:
It is confusing to return integer (row) as a boolean. This code might be easier to read as 

```
makeFlatVector({false});
```

It is still puzzling to the reader why making a vector of a single boolean? Would it be more straightforward to make a vector of a few integers?

```
makeFlatVector({1, 2, 3, 4, 5});
```